src/js/themes/base.d.ts

```diff
@@ -384,7 +384,7 @@ export interface ThemeType {
       primary?: ButtonKindType;
       secondary?: ButtonKindType;
     };
-    disabled?: ButtonKindType;
+    disabled?: ButtonKindType & { opacity?: OpacityType };
```

Comment:
Withdrawing the approval for now, should this be '|' instead of '&'?
can we add a test for it on the Button src/js/components/Button/stories/typescript/Custom.tsx story to make sure it works as expected?

Reply:
I think that it should be the `&`. An intersection makes more sense to me than a Union here because the button disabled will have button type and opacity, not one or the other. A test has been added in the most recent push

Reply:
In your example, you are not using ButtonKindType and you are only using `opacity`.

Reply:
Fixed in the most recent push